x-pack/plugin/logsdb/build.gradle

```diff
@@ -42,3 +42,13 @@ tasks.named("javaRestTest").configure {
 tasks.named('yamlRestTest') {
   usesDefaultDistribution()
 }
+
+tasks.named("yamlRestTest") {
+  if (buildParams.isSnapshotBuild() == false) {
+    systemProperty 'tests.rest.blacklist', [
+      "resources/rest-api-spec/test/60_synthetic_source_recovery.yml"
```

Comment:
Looking at other usages, I think this should just be: '60_synthetic_source_recovery/*'?